pkg/model/employees.go

```diff
@@ -24,47 +24,55 @@ type Employee struct {
 	BaseModel
 
 	// basic info
-	FullName                string
-	DisplayName             string
-	TeamEmail               string
-	PersonalEmail           string
-	Avatar                  string
-	PhoneNumber             string
-	Address                 string
-	MBTI                    string
-	Gender                  string
-	Horoscope               string
-	PassportPhotoFront      string
-	PassportPhotoBehind     string
-	IdentityCardPhotoFront  string
-	IdentityCardPhotoBehind string
-	DateOfBirth             *time.Time
+	FullName               string     `gorm:"column:full_name"`
```

Comment:
I think we don't need to use all gorm tags for all attributes. Only use for the one which cannot be recognized by ORM

Reply:
there are some field the gorm cannot recognized, so I used this solution for the whole structure to make things more consistent

Reply:
well we can consider to change it in another PR for all model